Sources/Quick/QuickSpec.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+// NOTE: This file is not intended to be included in the Xcode project. It
+//       is picked up by the Swift Package Manager during its build process.
+
+public class QuickSpec: XCTestCase, XCTestCaseProvider {
+    public func spec() {}
+
+    public required init() {}
+
+    public var allTests : [(String, () -> Void)] {
```

Comment:
Hmm, we're not the only ones with this problem: a [recent email](https://lists.swift.org/pipermail/swift-corelibs-dev/Week-of-Mon-20160111/000357.html) to the swift-corelibs-dev mailing list explained which snapshot was necessary to build swift-corelibs-foundaiton.

Until someone comes up with a little badge to put in the README, we *may* want to document somewhere which version of swift-corelibs-xctest we're targeting. Perhaps in the docblock to `QCKMain()`?